bin/test/game_spec.rb

```diff
@@ -0,0 +1,43 @@
+require './lib/game'
+
+describe Game do
+ 
+  before(:all) do
+    @player1 = Player.new("david", "X")
+    @player2 = Player.new("chibuzor", "O")
+    @current_player = @player1
+    @end_game = false
+    @board = Board.new
+  end
+
+   describe "#switch_player" do
+   game = Game.new
+   
+    it "should change to player2 if current player is player1" do 
+      expect(game.switch_player).to eql(game.get_player2)
+    end
+  end
+
+  describe "#choose_cell" do
+
+    game = Game.new
+    it "converts player input to zero-index based number" do 
+      expect(game.choose_cell(9)).to eql(8)
```

Comment:
We can improve these tests by looking at other cases too. 
What happens when the input is greter than maximum or less than minimum?